src/AppBundle/Service/EventProcessor.php

```diff
@@ -457,7 +457,7 @@ private function createOrUpdateEventStat($metric, $value, $offset = null)
     /**
      * Persist an EventWikiStat with given metric and value, or update the
      * existing one, if present.
```

Comment:
This might be unrelated, but if you change the parameter from "EventWikiStat" to "EventWiki" does that affect the purpose of the method? The documentation says "persiste an EventWikiStat..." -- should that change to "EventWiki" instead?

Reply:
This method takes an EventWiki and creates and persists an EventWikiStat for it. I've updated the doc block to make this more clear.